src/components/TypeAhead.tsx

```diff
@@ -78,8 +78,9 @@ class TypeAhead extends React.PureComponent<TypeaheadProps, TypeaheadState> {
           })
         }
         disabled={!showSuggestions}
+        className={className}
       >
-        <div className={cx(wrapper, className)} ref={this.typeAheadRef}>
+        <div className={cx(wrapper)} ref={this.typeAheadRef}>
```

Comment:
why even is this div ref'ed
typeAheadRef does not seem to be used. is it exported as an API?
I don't think this div should even be created.